app/web/components/UserSummary.tsx

```diff
@@ -131,13 +147,15 @@ export default function UserSummary({
         secondary={
           <>
             {!nameOnly && (
-              <Typography
-                color="textSecondary"
-                variant="body1"
-                noWrap={nameOnly}
-              >
-                {!user ? <Skeleton /> : user.city}
-              </Typography>
+              <Tooltip title={cityValue} arrow placement="top">
+                <Typography
+                  color="textSecondary"
+                  variant="body1"
+                  noWrap={nameOnly}
+                >
+                  {!user ? <Skeleton /> : <span>{displayCity}</span>}
```

Comment:
Logic here looks perfect! Upon looking at it, I think we can fit some more characters on the second line:

<img width="395" alt="Screenshot 2025-05-12 at 7 54 08 PM" src="https://github.com/user-attachments/assets/7a33dd8a-ca54-4197-8c3a-889cabd2caa8" />

Would you mind increasing the length for the displayCity to 50?